node/src/utils.ts

```diff
@@ -91,6 +91,25 @@ export function parseVector<Type>(
 	return array;
 }
 
+/**
+ * Parse flatbuffers vector of StringString into the corresponding array.
+ */
+export function parseStringStringVector(
+	binary: any, methodName: string
+): { key: string; value: string }[]
+{
+	const array: { key: string; value: string }[] = [];
+
+	for (let i=0; i<binary[`${methodName}Length`](); ++i)
```

Comment:
i = 0 with spaces as everywhere else. Not sure why ESLint doesn't complain.

Reply:
Eslint rule set.